### DISCUSSION TITLE
Why would you want to enable all FFlags by default?

### DISCUSSION THREAD

**Discussion by sasial-dev** [JohnnyMorganz/luau-lsp#36] [Q&A] [answered]
As found in https://github.com/JohnnyMorganz/luau-lsp/commit/3b07d67fc97c8e51ea589b361f0408ba5ab925ba.

**Comment by JohnnyMorganz** [comment, marked as answer]
Luau FFlags are typically for new features, and will most likely be committed to provided there are no bugs with them.
By default Luau's analyze and repl tool enable these FFlags, and I decided to as well - they normally provide benefits rather than drawbacks.
I made this an option though in case people did not want to enable them all by default, or they encountered any issues. If you disable `luau-lsp.fflags.enabledByDefault` and enable `luau-lsp.fflags.sync`, in theory you should have the exact same flags as Studio and should be able to replicate its behaviour
